manager/integration/tests/test_backing_image.py

```diff
@@ -421,3 +421,26 @@ def test_exporting_backing_image_from_volume():  # NOQA
         is the same as that of the 2nd volume.
     11. Do cleanup.
     """
+
+
+@pytest.mark.skip(reason="TODO") # NOQA
+@pytest.mark.backing_image  # NOQA
+def test_csi_snapshot_support(client, volume_name):  # NOQA
+    """
+    1. Test create CSI snapshot
+        1. Type is default or bak
```

Comment:
The type `default` should be already covered in the test `test_csi_volumesnapshot_basic`. We can add test cases only with `bak`, `snap` and `invalid`. 
refer tests in test_csi_snapshotter